manywheel/build_rocm.sh

```diff
@@ -311,7 +311,7 @@ PYTORCH_VERSION=$(cat $PYTORCH_ROOT/version.txt | grep -oP "[0-9]+\.[0-9]+\.[0-9
 if [ ${PYTORCH_VERSION%%\.*} -ge 2 ]; then
     if [[ $(uname) == "Linux" ]] && [[ "$DESIRED_PYTHON" != "3.12" || $(ver $PYTORCH_VERSION) -ge $(ver 2.4) ]]; then
 	# Triton commit got unified in PyTorch 2.5
-	if [[ $(ver $PYTORCH_VERSION) -ge $(ver 2.5) ]]; then
+	if [[ $(ver $PYTORCH_VERSION) -ge $(ver 2.5) && $(ver "$DESIRED_PYTHON") -lt $(ver "3.13")]]; then
```

Comment:
This is not where the condition needs to go. The py3.13 condition is basically to decide whether pytorch should have a python dependency on triton or not. For PyTorch2.5 and older, py3.13 was not supported for triton, so pytorch shouldn't have a py3.13 dependency on triton, but for PyTorch2.6 and newer, it can.

Reply:
This is the line that needs to conditionally add the `and python_version < '3.13'` part only for PyTorch2.5 and older: https://github.com/ROCm/builder/pull/68/files#diff-7096082d567b603ade9f800a8aad99eb4ec57ee15802d9dd4b4457f3a6609942R321